docs/reference/classes/LedgerLivePlatformSDK.md

```diff
@@ -236,15 +236,16 @@ The account selected by the user
 
 #### Defined in
 
-[LedgerLivePlatformSDK/index.ts:281](https://github.com/LedgerHQ/live-app-sdk/blob/5608a83/src/LedgerLivePlatformSDK/index.ts#L281)
+[LedgerLivePlatformSDK/index.ts:282](https://github.com/LedgerHQ/live-app-sdk/blob/db29f50/src/LedgerLivePlatformSDK/index.ts#L282)
 
 ___
 
 ### signMessage
 
 ▸ **signMessage**(`accountId`, `message`): `Promise`<`string`\>
 
-Let the user sign the provided message through Ledger Live. In Ethereum context, this is a [EIP-191 message](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md)
+Let the user sign the provided message through Ledger Live.
+In Ethereum context, this is a [EIP-191 message](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md) or a [EIP-712 message](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md)
```

Comment:
if you want to avoid rebuilding the doc
```suggestion
In Ethereum context, this is an [EIP-191 message](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md) or an [EIP-712 message](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md)
```